handlers/handlers_test.go

```diff
@@ -171,6 +175,66 @@ func TestUnitHandlers(t *testing.T) {
 				So(w.Code, ShouldEqual, http.StatusInternalServerError)
 			})
 		})
+
+		Convey("test Calendar (calendar/releasecalendar)", func() {
+			mockSearchClient := NewMockSearchAPI(mockCtrl)
+			url := "/calendar/releasecalendar/test"
+			router.HandleFunc(url, ReleaseCalendarICSEntries(*mockConfig, mockSearchClient))
+
+			Convey("it returns 200 when the ICS file is generated successfully", func() {
+				r := sitesearch.ReleaseResponse{
+					Releases: []sitesearch.Release{
+						{URI: url,
```

Comment:
We could also have test cases with multiple releases

Reply:
what does it add?

Reply:
I think it's a good idea testing multiple elements when working with lists so to ensure that we are not just using the first element but can deal with several ones

Reply:
Okay will do, mind you 0 and 1 in a list should generally be good if it's just a loop over a list?